pyrefinebio/config.py

```diff
@@ -35,26 +61,22 @@ def __new__(cls):
                 with open(cls.config_file) as config_file:
                     config = yaml.full_load(config_file) or {}
 
-            cls.token = config.get("token", "")
-            cls.base_url = os.getenv("BASE_URL") or config.get("base_url") or "https://api.refine.bio/v1/"
+            cls.token = os.getenv("RB_TOKEN") or config.get("token", "")
```

Comment:
I think `REFINEBIO_TOKEN` would be better. It's more verbose, but it's also more clear and easier to guess.